src/clarity-angular/select/providers/select.service.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Injectable, Optional, SkipSelf} from "@angular/core";
+import {Observable} from "rxjs/Observable";
+import {Subject} from "rxjs/Subject";
+
+import {Option} from "./../option";
+
+@Injectable()
+export class RootSelectService {
+    private _changes: Subject<boolean> = new Subject<boolean>();
+    private _input: Subject<string> = new Subject<string>();
+    private _specialKey: Subject<string> = new Subject<string>();
+    private _highlightedChange: Subject<Option> = new Subject<Option>();
+    public _highlighted: Option;
+    public _selected: Option;
+    private _selectedChange: Subject<Option> = new Subject<Option>();
+
+
+    get changes(): Observable<boolean> {
+        return this._changes.asObservable();
```

Comment:
Actually, I just realized you do not use this `_changes` or `changes` Observable anymore.